PWGCF/FemtoUniverse/Tasks/femtoUniversePairTaskTrackV0Extended.cxx

```diff
@@ -145,20 +152,15 @@ struct femtoUniversePairTaskTrackV0Extended {
   HistogramRegistry registryMCtruth{"MCtruthHistos", {}, OutputObjHandlingPolicy::AnalysisObject, false, true};
   HistogramRegistry registryMCreco{"MCrecoHistos", {}, OutputObjHandlingPolicy::AnalysisObject, false, true};
 
+  TH1* plocalEffp1 = 0;
```

Comment:
Please, please don't use raw pointers. You could use at least `std::shared_ptr`.

If you need to use files produced elsewhere, you should move that into a postprocessing macro outside the O2 analysis task. Loading efficiency histos from outside won't work on Hyperloop.

Reply:
Thanks a lot. I've swapped the raw pointers with `unique_ptr`.

For the efficiency loading, CCDB version will be added later. Loading locally works for me for now. I'm including it here in the commit so that it won't go through merge conflicts in the future - there are others from our group planning to extend this task.